### PR TITLE
feat(CORSO-33): view all reviews for customer

### PR DESCRIPTION
**Pull request by Mila5847**
JIRA: link to jira ticket
Ticket: CORSO-33
Link: https://champlainsaintlambert.atlassian.net/browse/CORSO-33
## Context:
This ticket is about allowing a customer to see their previously posted reviews through their profile.
## Changes
- "View All Reviews of Customer" functionality
- "View All Reviews of Customer" front-end
- "View All Reviews of Customer" testing
- "View All Reviews of Customer" E2E
-  Fixed broken CSS
### Before
No UI
### After
image
## Dev notes (Optional)
- Specific technical changes that should be noted
## Linked pull requests (Optional)
- pull request link

### CODE REVIEW

**Review by DenisaNicoletaH — APPROVED** [review]
I have not any comments to leave since I did not see anything that should've brought to your attention. I am aware that some of your code is also some bug fixes so I have nothing to say other than good job!

File: corso_website_springboot/src/test/java/com/corso/springboot/Review_Subdomain/presentationlayer/ReviewControllerTest.java [hunk 1]
```diff
@@ -9,17 +8,24 @@
...
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.oidcLogin;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @SpringBootTest(webEnvironment = RANDOM_PORT)
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
 @AutoConfigureWebClient
-@AutoConfigureMockMvc(addFilters = false)
+@AutoConfigureMockMvc(addFilters = true)
```
Comment (KarinaSofia) [inline corso_website_springboot/src/test/java/com/corso/springboot/Review_Subdomain/presentationlayer/ReviewControllerTest.java:28]: Why switch this from false to true?

**Review by KarinaSofia — APPROVED** [review]
Great Work!

File: corso_website_springboot/src/test/java/com/corso/springboot/Review_Subdomain/presentationlayer/ReviewControllerTest.java (hunk 1 above)
Comment (Mila5847) [reply corso_website_springboot/src/test/java/com/corso/springboot/Review_Subdomain/presentationlayer/ReviewControllerTest.java:28]: I needed to add the filter to be able to test if an user logged as a customer can view their reviews. It is related with the @PreAuthorize("hasAnyAuthority('Customer')") and to make sure that the customer is authorized i needed this .with(csrf()) and .with(oidcLogin().idToken(i -> i.subject("auth0|65702e81e9661e14ab3aac89")).authorities(new SimpleGrantedAuthority("Customer")))

**Pull request merged**